.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
This must've been from a time before I knew to include these files. Would you mind adding a more complete one? Maybe [this](https://github.com/github/gitignore/blob/master/Haskell.gitignore). Also, please ensure your editor includes trailing newlines.